homeassistant/components/abode/sensor.py

```diff
@@ -7,20 +7,26 @@
 
 from jaraco.abode.devices.sensor import Sensor as AbodeSense
 from jaraco.abode.helpers import constants as CONST
+from jaraco.abode.helpers.constants import UNIT_CELSIUS, UNIT_FAHRENHEIT
```

Comment:
Side note: These constants have been inlined in the latest version of the library.